api/src/Entity/DayResponsible.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/*
+ * Copyright (C) 2011 Urban Suppiger
+ *
+ * This file is part of eCamp.
+ *
+ * eCamp is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * eCamp is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with eCamp.  If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Entfernen? Wenn @usu einverstanden ist natürlich ;)

Reply:
Sonst alternativ überall (c) eCamp Commiters einfügen.

Reply:
Wenn wir mit License Headers beginnen würde ich mir unbedingt eine Automation dafür wünschen. Wir sind bereits jetzt schon sehr schlecht darin, bei neu kopierten Vue Komponenten den Header Comment mit Doku anzupassen...

Reply:
Eine Lizenzinfo pro Repo reicht, wir brauchen das nicht in jedem file, rechtlich gesehen.
Ich wäre für löschen.